static/app/views/performance/trends/changeExplorer.tsx

```diff
@@ -0,0 +1,243 @@
+  trendFunction,
+  trendView,
+  statsData,
+  isLoading,
+  organization,
+  projects,
+  trendParameter,
+  location,
+}: PerformanceChangeExplorerProps) {
+  return (
+    <DetailPanel detailKey={!collapsed ? transaction.transaction : ''} onClose={onClose}>
+      {!collapsed && (
+        <PanelBodyWrapper>
+          <ExplorerBody
+            transaction={transaction}
+            trendChangeType={trendChangeType}
+            trendFunction={trendFunction}
+            trendView={trendView}
+            statsData={statsData}
+            isLoading={isLoading}
+            organization={organization}
+            projects={projects}
+            trendParameter={trendParameter}
+            location={location}
+          />
+        </PanelBodyWrapper>
+      )}
+    </DetailPanel>
+  );
+}
+
+function ExplorerBody(props: ExplorerBodyProps) {
+  const {
+    transaction,
+    trendChangeType,
+    trendFunction,
+    trendView,
+    trendParameter,
+    isLoading,
+    location,
+    organization,
+  } = props;
+  const breakpointDate = transaction.breakpoint
+    ? moment(transaction.breakpoint * 1000).format('ddd, DD MMM YYYY HH:mm:ss z')
+    : '';
+
+  const start = moment(trendView.start).format('DD MMM YYYY HH:mm:ss z');
+  const end = moment(trendView.end).format('DD MMM YYYY HH:mm:ss z');
+  return (
+    <Fragment>
+      <Header transaction={transaction} trendChangeType={trendChangeType} />
+      <div style={{display: 'flex', gap: space(4)}}>
+        <InfoItem
+          label={
+            trendChangeType === TrendChangeType.REGRESSION
+              ? t('Regression Metric')
+              : t('Improvement Metric')
+          }
+          value={trendFunction}
+        />
+        <InfoItem label={t('Start Time')} value={breakpointDate} />
+      </div>
+      <GraphPanel data-test-id="pce-graph">
+        <strong>{`${trendParameter.label} (${trendFunction})`}</strong>
+        <ExplorerText color={theme.gray300} margin={`-${space(3)}`}>
+          {trendView.statsPeriod
+            ? DEFAULT_RELATIVE_PERIODS[trendView.statsPeriod] ||
+              getArbitraryRelativePeriod(trendView.statsPeriod)[trendView.statsPeriod]
+            : `${start} - ${end}`}
+        </ExplorerText>
+        <Chart
+          query={trendView.query}
+          project={trendView.project}
+          environment={trendView.environment}
+          start={trendView.start}
+          end={trendView.end}
+          statsPeriod={trendView.statsPeriod}
+          disableXAxis
+          disableLegend
+          neutralColor
+          {...props}
+        />
+      </GraphPanel>
+      <MetricsTable
+        isLoading={isLoading}
+        location={location}
+        transaction={transaction}
+        trendFunction={trendFunction}
+        trendView={trendView}
+        organization={organization}
+      />
+    </Fragment>
+  );
+}
+
+function InfoItem({label, value}: {label: string; value: string}) {
+  return (
+    <div style={{float: 'left'}}>
```

Comment:
Do you need `float: 'left'`? I removed it locally and I'm not really seeing any difference